src/styles/cards.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  &__img {
+    margin-left: 3px;
+    width: 160px;
+    height: 134px;
+    box-sizing: border-box;
+  }
+
+  &__description {
+    margin: 0;
+    padding-top: 40px;
+    font-size: 12px;
+    height: 36px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__code {
+    justify-content: space-between;
+    margin: 0;
+    padding-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $not-important-text;
+  }
+
+  &__rate {
+    margin-left: 2px;
+    padding-top: 17px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    text-align: center;
+    padding-top: 23px;
+
+    &-text {
+      margin: 0;
+      font-size: 12px;
+      line-height: 18px;
+      color: $not-important-text;
+    }
+
+    &-sum {
+      margin: 0;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    padding-top: 16px;
+
+    &-buy {
+      font-family: Roboto, sans-serif;
+      background-color: $collor-button;
+      border: none;
+      border-radius: 5px;
+      color: white;
+      justify-content: center;
+      font-size: 14px;
+      line-height: 16px;
+      font-weight: 700;
+      text-transform: uppercase;
+      width: 166px;
+      height: 40px;
+      text-decoration: none;
```

Comment:
tip: try adding empty lines between similar css properties when there are a lot of them

Reply:
I'm not adding lines in or out some things, the lint is doing by itself, actually we had a few issues because of that. When the script runs "script mate + script lint" our LATEST lint will do it auto. I'm on it right now and already reported this.